pkg/storaged/utils.js

```diff
@@ -791,6 +815,9 @@ export function get_active_usage(client, path, top_action, child_action) {
         }
 
         function enter_unmount(block, location, is_top) {
+            const [, mount_point] = get_fstab_config_with_client(client, block);
```

Comment:
Seeing that `get_fstab_config_with_client` can return [], this leads to `mount_point` being undefined, it might be better to use `===` below. But looking at it, it should do the right thing currently.